go.mod

```diff
@@ -3,24 +3,25 @@ module github.com/honeycombio/honeytail
 go 1.17
 
 require (
-	github.com/go-sql-driver/mysql v1.7.1
+	github.com/go-sql-driver/mysql v1.8.1
 	github.com/honeycombio/dynsampler-go v0.6.0
 	github.com/honeycombio/gonx v1.3.1-0.20171118020637-f9b2468e9ef8
-	github.com/honeycombio/libhoney-go v1.22.0
+	github.com/honeycombio/libhoney-go v1.23.1
```

Comment:
minimum go version is now 1.19 with[ this libhoney update](https://github.com/honeycombio/libhoney-go/releases/tag/v1.23.0), should we label this PR as breaking and flag it in changelog?  release notes will pick it up automatically if you auto-generate.